reference/api/bindings_api.md

```diff
@@ -113,7 +113,7 @@ In order to deliver binding inputs, a POST call is made to user code with the na
 #### HTTP Request
 
 ```http
-POST http://localhost:<appPort>/<name>
+POST http://localhost:<appPort>/v1.0/<name>
```

Comment:
@youngbupark  @yaron2 would this binding callback need a version. My view is that it should not